src/averageAges.js

```diff
@@ -20,6 +20,13 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const isMan = (person) => (person.sex === 'm');
```

Comment:
such names for variables are usually used for a boolean value. Here better smth like checkIfMan